File: launchdarkly/ruby/src/PostDestinationExample.rb
require "json"
require "launchdarkly_client"

LaunchDarklyClient.configure do |config|
    config.api_key["Authorization"] = "YOUR_API_KEY"
end

destination_post = LaunchDarklyClient::DestinationPost.new
destination_post.kind = "google-pubsub"

begin
    response = LaunchDarklyClient::DataExportDestinationsApi.new.post_destination(
        "projectKey_string", # project_key
        "environmentKey_string", # environment_key
        destination_post,
    )

    p response
rescue LaunchDarklyClient::ApiError => e
    puts "Exception when calling DataExportDestinationsApi#post_destination: #{e}"
end

File: launchdarkly/python/src/PostDestinationExample.py
import json
from datetime import date, datetime
from pprint import pprint

from launchdarkly_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    api_key={"ApiKey": "YOUR_API_KEY"},
)

with ApiClient(configuration) as api_client:
    destination_post = models.DestinationPost(
        kind="google-pubsub",
    )

    try:
        response = api.DataExportDestinationsApi(api_client).post_destination(
            project_key=None,
            environment_key=None,
            destination_post=destination_post,
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling DataExportDestinationsApi#post_destination: %s\n" % e)
